dc/core/CMakeLists.txt

```diff
@@ -47,14 +47,10 @@ endif()
 list(APPEND DCCORE_PUBLIC_HEADERS
   ${COMMON_INCLUDES}
   types.h
-  Content.h
   ContentFactory.h
   ContentLoader.h
   ContentType.h
   ContentInteractionDelegate.h
-  ContentWindow.h
```

Comment:
non-public headers should go to DCCORE_HEADERS. With your change, they are not tracked anymore by CMake, and I don't see them anymore in QtCreator :)
Or are those headers somewhere else referenced?